website/thaliawebsite/settings/settings.py

```diff
@@ -247,6 +256,7 @@
 
 # Compressor settings
 COMPRESS_ENABLED = True
+COMPRESS_OFFLINE = not DEBUG
```

Comment:
This line can be removed as it's the default value

Reply:
That is not true, the default value is `False`: https://django-compressor.readthedocs.io/en/stable/settings/#django.conf.settings.COMPRESS_OFFLINE